cloudformation/custom_resource/cluster.yaml

```diff
@@ -81,7 +81,20 @@ Resources:
               - events:PutTargets
               - events:RemoveTargets
             Resource: !Sub arn:${AWS::Partition}:events:${AWS::Region}:${AWS::AccountId}:rule/*
-
+  S3Policy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Sid: S3Policy
+            Effect: Allow
+            Action: s3:*
```

Comment:
Can we scope this down?

Reply:
We could. I didn't do it because the resource limited to a single bucket

Reply:
Security raises an eye of they see this (regardless)

Reply:
Done